tools/stub_generator/cuda.json

```diff
@@ -7,6 +7,8 @@
    "not_found_error":"CUDA_ERROR_SHARED_OBJECT_SYMBOL_NOT_FOUND",
    "functions": {
       "cuInit": {},
+      "cuGetProcAddress": {},
+      "cuGetProcAddress_v2": {},
```

Comment:
depending on CUDA header versions, we need either the former or the latter (starting with CUDA 12.0).